### ISSUE TITLE
Using xlash framework for automated testing using postman

### ISSUE THREAD

**Issue by prabhukirannarava** [xlash/framework_for_automated_TC#1]
Hi,

can you please help me in using your framework for automated testing of my APIs ?

Preferablly with any example.

**Comment by xlash** [comment]
Yes for sure. Have you loaded up the framework javascripts first and installed all files already?

**Comment by prabhukirannarava** [comment]
Hi,

I did not have framework java scripts loaded. And that is where I am facing a problem.
Requesting you share me step by step guide to install the same.
Thanks for your help!!

**Comment by xlash** [comment]
1 - You need to install the files in the Postman plugin repository
2- All files with extension .patch, follow instructions inside to be able to patch correctly the original Postman files.
3- Load postman afterwards. If framework doesn't load, please post javascript error message you are getting.

**Comment by prabhukirannarava** [comment]
Can you elaborate point 1 and 2 ?

I tried finding post man plug in repository. But I could not find it.
Can you please let me know in which path this repository will be available ?

i tried to find it in below path..
C:\Program Files (x86)\Google\Chrome

but did not get it. :(

It would be of help if you can show me with an example test.

Thanks a lot for your help.!!!

**Comment by xlash** [comment]
When you go to Chrome Options, Plugins, you can expand by clicking developper mode. You'll see a name like fhbjgbiflinjbdggehcddcbncdddomop next to the Postman plugin. Locate this folder in your computer by searching. This will be the postman repository.

This should be in your User/AppData folder.

In the repository, you'll find the files like requester.js, which you can apply manually the requester.js.patch afterwards. Best is you locate the files first, and try manually patching them by following the instructions.
